apps/web/src/services/dashboards/widgets/cost-widgets/aws-data-transfer-cost-trend/AWSDataTransferCostTrendWidget.vue

```diff
@@ -113,58 +111,76 @@ const { widgetState, widgetFrameProps, widgetFrameEventHandlers } = useWidget(pr
 const state = reactive({
     loading: true,
     data: null as FullData | null,
-    fieldsKey: computed<'cost_sum'|'usage_quantity_sum'>(() => (selectedSelectorType.value === 'cost' ? 'cost_sum' : 'usage_quantity_sum')),
+    fieldsKey: computed<'cost'|'usage_quantity'>(() => (selectedSelectorType.value === 'cost' ? 'cost' : 'usage_quantity')),
     chartData: computed<ChartData[]>(() => {
         if (!state.data?.results) return [];
-        const dataKey = state.fieldsKey;
+
         const chartData: ChartData[] = getRefinedXYChartData<Data, ChartData>(state.data.results, {
             groupBy: USAGE_TYPE_VALUE_KEY,
-            arrayDataKey: dataKey,
+            arrayDataKey: 'value_sum',
             categoryKey: DATE_FIELD_NAME,
             valueKey: 'value',
-            additionalIncludeKeysFromParent: ['usage_unit'],
         });
         return sortBy(chartData, widgetState.dateRange, DATE_FIELD_NAME);
     }),
     tableFields: computed<Field[]>(() => {
-        const _textOptions: Field['textOptions'] = {
-            type: state.fieldsKey === 'cost_sum' ? 'cost' : 'number',
+        const _textOptions: Field['textOptions'] = state.fieldsKey === 'cost' ? {
+            type: 'cost',
+        } : {
+            type: 'usage',
+            unitPath: 'usage_unit',
         };
 
-        const refinedFields = getWidgetTableDateFields(widgetState.granularity, widgetState.dateRange, _textOptions, state.fieldsKey);
+        const refinedFields = getWidgetTableDateFields(widgetState.granularity, widgetState.dateRange, _textOptions, 'value_sum');
```

Comment:
note: `unitPath` is the path to get the value of unit when refining value of each cell.